icu4c/source/test/intltest/dtifmtts.cpp

```diff
@@ -1810,7 +1810,7 @@ void DateIntervalFormatTest::testTicket20707() {
     IcuTestErrorCode status(*this, "testTicket20707");
 
     const char16_t timeZone[] = u"UTC";
-    Locale locales[] = {"en-u-hc-h24", "en-u-hc-h23", "en-u-hc-h12", "en-u-hc-h11"};
+    Locale locales[] = {"en-u-hc-h24", "en-u-hc-h23", "en-u-hc-h12", "en-u-hc-h11", "en"};
```

Comment:
Can you also make a test with an invalid value like `en-u-hc-h25`?

Reply:
Done.